docs/admin/admission-controllers.md

```diff
@@ -142,6 +143,15 @@ applies a 0.1 CPU requirement to all Pods in the `default` namespace.
 
 See the [limitRange design doc](../design/admission_control_limit_range.md) and the [example of Limit Range](limitrange/) for more details.
 
+### InitialResources (experimental)
+
+This plug-in observes pod creation requests. For each container that does not specify compute resource requests or limits,
```

Comment:
I would phrase as follows:

If a container omits compute resource requests and limits, then the plug-in auto-populates a compute resource request based on historical usage of containers running the same image.